samples/exceptions/ExceptionFileNotExist.java

```diff
@@ -0,0 +1,8 @@
+package exceptions;
+
+public class ExceptionFileNotExist extends RuntimeException {
```

Comment:
It is better to name exception class with "Exception" word in the end, e.g. FileNotExistException.